fix ordine ignorato

Symon02 opened the pull request:
ho aggiunto l'if che permette di elaborare anche i panini senza l'ingrediente principale

Symon02 committed:
Add exixting file

dbarasti reviewed (changes requested):
@Symon02 anchè ritornare 5 (riga 21), il `basePrice` deve diventare 5. Altrimenti in questo caso la funzione `computeIngredientsPrice` non viene mai chiamata. Please fix this

Symon02 committed:
fixed price error 

Ora, nel caso il panino sia senza carne, viene calcolato correttamente il prezzo, tenendo conto delle farciture al suo interno

Symon02 committed:
Delete Calculator.js

Symon02 committed:
price error fix

Ora, nel caso del panino senza ingrediente principale, viene calcolato correttamente il prezzo, tenendo conto delle farciture presenti
